cluster-1.15.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: martina-1-15-9
```

Comment:
Was this file checked in accidentally? If not, you may want to move it to `examples` and to change the cluster's name.